Tools/cases_generator/generate_cases.py

```diff
@@ -252,6 +252,7 @@ class InstructionFlags:
     HAS_CONST_FLAG: bool
     HAS_NAME_FLAG: bool
     HAS_JUMP_FLAG: bool
+    NO_EXCEPTION_FLAG: bool
```

Comment:
Negative flags tend to lead to confusing double-negative code constructs. So I would prefer for the flag to be `CAN_RAISE_FLAG` than `NO_EXCEPTION_FLAG`, and similar throughout the PR.